docs/release-notes/8.6.asciidoc

```diff
@@ -0,0 +1,92 @@
+* Updates Osquery schema to v5.5.1 ({pull}144090[#144090]).
+* Improves the UX for managing {ml} jobs while managing {ml} rules ({pull}144080[#144080]).
+* Enables you to run {ml} jobs from the Notable Anomalies table ({pull}142861[#142861]).
+* Adds a guided onboarding flow to help you create your first rules ({pull}144016[#144016], {pull}143598[#143598]).
+* Updates the take action UI for charts on the Hosts, Users, and Network pages ({pull}138369[#138369]).
+* Adds a *Respond* button to the Host Details page for hosts with an agent installed ({pull}143988[#143988]).
+* Allows you to add up to three new terms to New Terms rule queries, enabling you to create alerts when multiple new terms appear in the same event ({pull}143943[#143943]).
+* Allows you to launch Timeline from the Entity Analytics dashboard by clicking alert counts ({pull}143841[#143841]).
+* Adds missing TLP Marking badges to the Indicators table and Indicator details page ({pull}143431[#143431]).
+* Ensures the empty state of the Indicators page does not appear when any threat intelligence integrations are installed ({pull}143328[#143328]).
+* Turns the anomalies count on the Entity Analytics dashboard into a link that jumps down the page to the Anomalies table ({pull}143085[#143085])
+* Pre-selects the `threat` category when you open the Fields browser ({pull}142698[#142698]).
+* Adds a `copy to clipboard` action for indicators throughout the Threat Intelligence plugin ({pull}142675[#142675]).
+* Adds a `User risk classification` column to the Users table ({pull}142610[#142610]).
+* Adds a label to the Indicators page that states when it was last updated ({pull}142560[#142560]).
+* Specifies that links from the Threat Intelligence plugin to the Integrations page should open the Threat Intelligence integrations category ({pull}142538[#142538]).
+* Enables full-screen mode on the Indicators table ({pull}142519[#142519]).
+* Implements the standard searchbar and date picker on the Threat Intelligence page ({pull}142336[#142336]).
+* Updates the design of the Rule Exceptions page ({pull}142289[#142289]).
+* Displays comments for expanded items in the Action history page ({pull}141938[#141938]).
+// Items below this line were labeled as "bugfixes" rather than "enhancements"
+* Replaces the *Run job* button with a *Stop job* button when the job is running ({pull}146407[#146407]).
+* Fixes a bug that prevented you from editing an exception while adding a comment to it from the Rules details flyout ({pull}145575[#145575]).
+* Fixes a bug that could cause rule previews for New Terms rules to fail ({pull}145707[#145707]).
+* Fixes a bug that could cause a `Page not found` error when you navigated to a shared exception list ({pull}145833[#145833]).
+* Fixes a bug with the loading indicator that appears when bulk actions are pending ({pull}145905[#145905]).
+* Fixes a bug with the display of the count of linked rules for shared exception lists ({pull}145976[#145976]).
```

Comment:
```suggestion
* Fixes a bug with the linked rules count for shared exception lists ({pull}145976[#145976]).
```

Reply:
@WafaaNasr would you mind letting us know if this summary is accurate?

Reply:
It is accurate! :) Thanks Nastasha and @benironside 
Great work!